server/src/main/java/com/github/animelist/animelist/AnimeListRuntimeWiringConfigurer.java

```diff
@@ -1,15 +1,34 @@
 package com.github.animelist.animelist;
 
+import com.github.animelist.animelist.model.ratingsystem.ContinuousRatingSystem;
+import com.github.animelist.animelist.model.ratingsystem.DiscreteRatingSystem;
 import graphql.scalars.ExtendedScalars;
 import graphql.schema.idl.RuntimeWiring;
+import graphql.schema.idl.TypeRuntimeWiring;
 import org.springframework.graphql.execution.RuntimeWiringConfigurer;
 import org.springframework.stereotype.Component;
 
+import static graphql.schema.idl.TypeRuntimeWiring.newTypeWiring;
+
 @Component
 public class AnimeListRuntimeWiringConfigurer implements RuntimeWiringConfigurer {

```

Comment:
We need type resolver for the interface "RatingSystem" that was defined in the schema